<launch>
  <node pkg="pcl_obstacle_detection" type="homing_filter_node" respawn="false" name="homing_filter_node" output="screen">
    <rosparam
      command="load"
      file="$(find pcl_obstacle_detection)/config/config.yaml"
      subst_value="true"
    />
  </node>
</launch>
